tla/tlc.sh

```diff
@@ -3,7 +3,14 @@
 # Licensed under the Apache 2.0 License.
 # Original License below
 # Adapted from: https://github.com/pmer/tla-bin
-exec java -cp tla2tools.jar:CommunityModules-deps.jar tlc2.TLC "$@" -workers auto
+
+TLC_OPTIONS=()
+
+if [ "${CI}" ] || [ "${SYSTEM_TEAMFOUNDATIONCOLLECTIONURI}" ]; then
+    JVM_OPTIONS=("-Dtlc2.TLC.ide=Github" "-Dutil.ExecutionStatisticsCollector.id=be29f6283abeed2fb1fd0be898bc6601")
+fi
+
+exec java -XX:+UseParallelGC -Dtlc2.tool.impl.Tool.cdot=true "${JVM_OPTIONS[@]}" -cp tla2tools.jar:CommunityModules-deps.jar tlc2.TLC "$@" -workers auto -lncheck final "${TLC_OPTIONS[@]}"
```

Comment:
This change seems to have dropped `-tool`, not sure if @lemmy was using this

Reply:
Happy to restore that in the non-CI context, also not sure if it's used. It's always possible to pass additional arguments of course, this is only setting defaults.

Reply:
With `-tool` one can import the CI output in the Toolbox and VSCode.

Reply:
@lemmy I find the output very verbose and difficult to read, is there a way to generate `-tool` output to file separately (which we can publish and make available)?

Reply:
You would have to use tee and grep since there is nothing in TLC. Alternatively, let's remove the -tool option, as reproducing a violation is fast with our small models.